rest-service/manager_rest/manager_elasticsearch.py

```diff
@@ -52,12 +54,20 @@ def build_request_body(filters=None, pagination=None, skip_size=False,
                         and values are the range limits of that field
         :return: An elasticsearch Query DSL body.
         """
+
+        def _escape_reserved_es_chars(val):
+            reserved_chars_regex = \
+                '([\(\)\{\}\+\-\=\>\<\!\[\]\^\"\~\*\?\:\\/]|&&|\|\|\s)'
+            return re.sub(reserved_chars_regex, r'\\\1', val)
+
         def _build_query_match_condition(k, val_list):
+            val_list = \
+                [_escape_reserved_es_chars(val) for val in val_list]
```

Comment:
in wildcard, the key is part of the user query, rather than part of the structured query